### DISCUSSION TITLE
Maui is bug ridden

### DISCUSSION THREAD

**Discussion by williambuchanan2** [dotnet/maui#12237] [General]
Is it just me (and everyone I know who is using Maui), or is Maui just full of bugs?

Just getting so frustrated with the incredible number of problems we have to work around every day. Finally get it working, and then the next morning it won't build/deploy/etc - even if nothing has changed.

As for deploying from Azure Devops - good luck with that!

Would be good if Microsoft tried to do something about this rather than moving on to the next shiny object...

**Comment by Majestic-Mustang** [comment]
They need to invest heavily on automated testing in MAUI repo before they can achieve true stability. 
It would save them from integration bugs and help them refine the code quickly. 

They need to do this: https://github.com/dotnet/maui/discussions/11354
